app/src/main/java/com/mstudio/superheroarch/remotedatasource/api/RickAndMortyApi.kt

```diff
@@ -17,4 +17,8 @@ interface RickAndMortyApi {
 
     @GET("episode/{id}")
     suspend fun getSingleEpisode(@Path("id") id: Int): Response<EpisodeRemoteEntity>
+}
+
+object RickAndMortyApiHelper {
+    fun create(): RickAndMortyApi = RetrofitInstance.retrofit().create(RickAndMortyApi::class.java)
```

Comment:
where do you set the base URL of this API? I cannot find it in this PR.

Reply:
I have it is this other class
```object RetrofitInstance {
    private const val BASE_URL = "https://rickandmortyapi.com/api/"

    fun retrofit(baseUrl: String = BASE_URL): Retrofit {
        return Retrofit.Builder()
            .baseUrl(baseUrl)
            .addConverterFactory(GsonConverterFactory.create())
            .build()
    }
}```

Reply:
you should then be consistent and implement the same solution in both API. It is not good to have what you describe and then something different in `TheMovieDbApi.kt`